client/src/components/ContainerReviews.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const ContainerReviews = props => (
+  <div>
+    <h3> Reviews Container </h3>
```

Comment:
One minor nit picky thing is indention. Here it looks like you used two spaces where in client/src/components/ContainerAverageStars.jsx line 7 and 8 it appears to be a tab equal to 4 spaces.

Reply:
Thanks for pointing that out, I'll take a look!  I'm trying to do what the linter suggests, sometimes it's a bit particular.

Reply:
Dude I had tabs!  Now fixed.  Thanks for catching man, definitely not nit picky, I appreciate you pointing it out!